src/box/index.cc

```diff
@@ -631,29 +629,18 @@ iterator_position(struct iterator *it, const char **pos, uint32_t *size)
 	return it->position(it, pos, size);
 }
 
-/** Keys for iterator position map. All the keys must be uint. */
-enum {
-	ITERATOR_POSITION_KEY = 0,
-};
+static const int iterator_pos_b64_opts = BASE64_NOPAD | BASE64_NOWRAP;
```

Comment:
Nit: Please use a define instead: a static variable is allocated on bss, which makes no sense in this case.